contracts/StakeLocker.sol

```diff
@@ -64,11 +64,8 @@ contract StakeLocker is StakeLockerFDT, Pausable {
     modifier canUnstake(address from) {
         IPool _pool = IPool(pool);
 
-        require(
-            (from != _pool.poolDelegate() && _pool.isPoolFinalized()) ||
-            !_pool.isPoolFinalized(),
-            "SL:STAKE_LOCKED"
-        );
+        // Pool cannot be finalized, but if it is, user cannot be the Pool Delegate
+        require(!_pool.isPoolFinalized() || from != _pool.poolDelegate(), "SL:STAKE_LOCKED");
```

Comment:
`(!A && B) || !C` is really `!C || !B` given [short-circuiting](https://docs.soliditylang.org/en/v0.6.11/types.html?highlight=short-circuiting#booleans).